IPython/html/templates/tree.html

```diff
@@ -76,6 +76,24 @@
           </div>
           <div id="notebook_list">
             <div id="notebook_list_header" class="row list_header">
+
+<div class="dropdown" id='tree-selector'>
+  <button class="btn btn-default btn-xs dropdown-toggle" type="button" id="tree-selector-btn" data-toggle="dropdown" aria-expanded="true">
+    <input type='checkbox'></input>
+    <i class='semicheck fa fa-minus'></i>
+    <span class="caret"></span>
+  </button>
+  <ul class="dropdown-menu" role="menu" aria-labelledby="tree-selector-btn">
+    <li role="presentation" class="select-all"><a role="menuitem" tabindex="-1" href="#">Select all</a></li>
+    <li role="presentation" class="select-notebooks"><a role="menuitem" tabindex="-1" href="#">Select notebooks</a></li>
+    <li role="presentation" class="select-running-notebooks"><a role="menuitem" tabindex="-1" href="#">Select running notebooks</a></li>
+    <li role="presentation" class="select-files"><a role="menuitem" tabindex="-1" href="#">Select files</a></li>
+    <li role="presentation" class="select-directories"><a role="menuitem" tabindex="-1" href="#">Select directories</a></li>
+    <li role="presentation" class="select-inverse"><a role="menuitem" tabindex="-1" href="#">Select inverse</a></li>
+    <li role="presentation" class="unselect-all"><a role="menuitem" tabindex="-1" href="#">Unselect all</a></li>
```

Comment:
I should nest this block too.  I was working in a non-nested state so I could easily keep track of what was new and what wasn't.